<ui version="4.0" >
 <class>TextureEditorClass</class>
 <widget class="QWidget" name="TextureEditorClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TextureEditor</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>405</width>
        <height>405</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Texture</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAction" >
     <property name="title" >
      <string> Action selection </string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QPushButton" name="moveButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Allow pan and zoom on the UV Space</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/selpan.png</normaloff>:/images/selpan.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="selectButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Allow the selection of faces inside a dragged area</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/selarea.png</normaloff>:/images/selarea.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4" >
       <widget class="QPushButton" name="connectedButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Allow the selection of connected component</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/selcon.png</normaloff>:/images/selcon.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6" >
       <widget class="QPushButton" name="vertexButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Allow the selection of vertexes inside a dragged area</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/selver.png</normaloff>:/images/selver.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="7" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="8" >
       <widget class="QPushButton" name="unify2Button" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Collapse a couple of vertexes</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/collapse.png</normaloff>:/images/collapse.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QPushButton" name="unifySetButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Choose and unify a set of disconnected edges</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/unify.png</normaloff>:/images/unify.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2" >
       <widget class="QPushButton" name="flipHButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Flip selected faces horizzontally</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/flipH.png</normaloff>:/images/flipH.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QPushButton" name="flipVButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Flip selected faces vertically</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/flipV.png</normaloff>:/images/flipV.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="6" >
       <widget class="QPushButton" name="invertButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Invert selected faces/vertexes</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/invertsel.png</normaloff>:/images/invertsel.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="8" >
       <widget class="QPushButton" name="cancelButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Clear selected faces/vertexes</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="meshlab.qrc" >
          <normaloff>:/images/selcanc.png</normaloff>:/images/selcanc.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOptimize" >
     <property name="title" >
      <string> Optimization </string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="maximumSize" >
         <size>
          <width>182</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title" >
         <string> Remap </string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="clampButton" >
           <property name="maximumSize" >
            <size>
             <width>70</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Clamp texture coordinates of of border</string>
           </property>
           <property name="text" >
            <string>Clamp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modulusButton" >
           <property name="maximumSize" >
            <size>
             <width>70</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Apply moudlus operator to texture coordinates out of border</string>
           </property>
           <property name="text" >
            <string>Modulus</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QGroupBox" name="groupBox_5" >
        <property name="title" >
         <string> Change Texture </string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="labelName" >
           <property name="text" >
            <string>---</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton" >
           <property name="maximumSize" >
            <size>
             <width>90</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string> Smooth texture coordinates </string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>9</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonWhole" >
           <property name="text" >
            <string>Whole model</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSelected" >
           <property name="text" >
            <string>Selected faces</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Iteration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxIteration" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>10</number>
           </property>
           <property name="value" >
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="smoothButton" >
           <property name="text" >
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>moveButton</tabstop>
  <tabstop>selectButton</tabstop>
  <tabstop>connectedButton</tabstop>
  <tabstop>vertexButton</tabstop>
  <tabstop>invertButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>clampButton</tabstop>
  <tabstop>modulusButton</tabstop>
  <tabstop>spinBoxIteration</tabstop>
  <tabstop>smoothButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="meshlab.qrc" />
 </resources>
 <connections/>
</ui>
